.release_templates/pytorch_release_images.yml

```diff
@@ -0,0 +1,34 @@
+---
+release_images:
+  1:
+    framework: "pytorch"
+    version: "1.6.0"
```

Comment:
nit: pytorch 1.6 isn't released yet, so it would make more sense to choose 1.5.0 (i.e. if we needed to use this today, we would have to change to a 1.5 version

Reply:
+1